lib/src/widgets/chat/ds_location_message_bubble.widget.dart

```diff
@@ -36,10 +36,13 @@ class DSLocationMessageBubble extends StatelessWidget {
       onTap: () async {
         final availableMaps = await MapLauncher.installedMaps;
 
-        await availableMaps.first.showMarker(
-          coords: Coords(latitude, longitude),
-          title: title ?? '',
-        );
+        if (double.tryParse(latitude) != null &&
```

Comment:
I would do something like that to reuse the returning value of ```double.tryParse```, instead of discarding it:

<img width="451" alt="image" src="https://github.com/takenet/blip-ds-flutter/assets/91232521/9ac97ef5-2e58-4715-b708-bdae0f83d867">